Modules/_ctypes/_ctypes.c

```diff
@@ -1351,7 +1351,7 @@ PyCArrayType_new(PyTypeObject *type, PyObject *args, PyObject *kwds)
     length = PyLong_AsLongAndOverflow(length_attr, &overflow);
     if (overflow) {
         PyErr_SetString(PyExc_OverflowError,
-                        "The '_length_' attribute is too large");
+                        "The '_length_' attribute does not fit in C long");
```

Comment:
all right. I would also add a `test_bad_length` test to Lib/ctypes/test/test_arrays.py.

by the way, according to https://docs.python.org/3.7/library/ctypes.html#ctypes.Array._length_,
_length_ must be positive (ISTM that it makes sense, as an array of length 0 is useless), but currently no error is raised in case _length_ == 0.
do you think this is OK?